src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -138,33 +138,19 @@ public void Add(ITagHelper tagHelper)
         }
 
         /// <summary>
-        /// Tracks the minimized HTML attribute.
-        /// </summary>
-        /// <param name="name">The minimized HTML attribute name.</param>
-        public void AddMinimizedHtmlAttribute(string name)
```

Comment:
Structure is more first class now and always provided, doesn't make sense to keep this.